README.md

```diff
@@ -36,6 +36,14 @@ LLM App reads a corpus of documents stored in S3 or locally, preprocesses them,
 - **User session and beta testing handling:** The query building process can be extended to handle user sessions and beta testing for new models.
 - **Code reusability for offline evaluation:** The same code can be used for static evaluation of the system.
 
+### Coming Soon:
+- Split the application into indexing and request-serving processes easily.
+- Add control for which model is used
+- Expand context doc selection with a graph walk
+- Model drift and monitoring setup
+- Model A/B testing support
```

Comment:
```suggestion
- Splitting the application into indexing and request-serving processes easily.
- Controlling which model is used.
- Expanding context doc selection with a graph walk.
- Model drift and monitoring setup.
- Model A/B testing support.
```
or `the ability to split ..." in the first case - depending on whether you will split it and it is going to be split by default or you are just going to give users the ability to split it.
I don't understand the point "Add control for which model is used" - don't we have this control now? Do you plan to move it to env variables or what?
I changed to text so that all points have the same grammatical form. However, feel free to disagree with the changes.

Reply:
> 



> or `the ability to split ..." in the first case - depending on whether you will split it and it is going to be split by default or you are just going to give users the ability to split it. I don't understand the point "Add control for which model is used" - don't we have this control now? Do you plan to move it to env variables or what? I changed to text so that all points have the same grammatical form. However, feel free to disagree with the changes.

Bullet points provided by Jan. Was gonna discuss with him when he get back. I have the same impression.

Reply:
Ok. So then maybe we have to merge it ;-)

Reply:
Or maybe let's apply the fixes? Maybe he provided general thoughts, not the exact form. If you want, choose your own fixes.